src/components/pages/Offers/Offer/OfferDetails/OfferForm/OfferForm.jsx

```diff
@@ -96,6 +97,8 @@ const OfferForm = ({
   const formRef = useRef(null)
   const [isLoading, setIsLoading] = useState(true)
 
+  let mandatoryFields = MANDATORY_FIELDS
```

Comment:
```suggestion
  let mandatoryFields = [...MANDATORY_FIELDS]
```
Sinon, les variables `mandatoryFields` et  `MANDATORY_FIELDS` pointent vers la même référence et le `push` mettra à jours les deux variables.